{"name": "Ошибка '409' при создании двух одинаковых курьеров", "status": "passed", "steps": [{"name": "Создаём двух одинаковых курьеров", "status": "passed", "start": 1747763407620, "stop": 1747763408094}], "start": 1747763407620, "stop": 1747763408095, "uuid": "7b0ebe71-cfeb-43ad-a4ef-9c4b277d8306", "historyId": "acd8d8e8195c89825fb471e3fc9b39be", "testCaseId": "acd8d8e8195c89825fb471e3fc9b39be", "fullName": "tests.test_creating_courier.TestCreatingCourier#test_error_when_creating_two_identical_couriers", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_creating_courier"}, {"name": "subSuite", "value": "TestCreatingCourier"}, {"name": "host", "value": "KOMPUTER"}, {"name": "thread", "value": "6036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_creating_courier"}]}